openmdao/recorders/recording_manager.py

```diff
@@ -1,12 +1,15 @@
 """
 RecordingManager class definition.
 """
-from six import iteritems
 import time
+from six import iteritems
 
 import numpy as np
 
-from openmdao.utils.mpi import MPI
+try:
+    from openmdao.utils.mpi import MPI
+except ImportError:
+    MPI = False
```

Comment:
Later in the file there is an 'if MPI is None' check, so the value of MPI should be set to None here.  None is the value we assign it elsewhere as well whenever MPI is not active or not installed.